src/sentry/static/sentry/app/locale.jsx

```diff
@@ -7,6 +7,7 @@ import _ from 'underscore';
 
 let LOCALE_DEBUG = false;
 
+let sessionStorage = window.sessionStorage;
```

Comment:
`window` is mocked, which means `sessionStorage` is undefined instead of a `ReferenceError` (`sessionStorage is not defined`).